curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-to-assign-variables-from-objects.english.md

```diff
@@ -19,7 +19,7 @@ You may read it as "get the field <code>x</code> and copy the value into <code>a
 
 ## Instructions
 <section id='instructions'>
-Use destructuring to obtain the average temperature for tomorrow from the input object <code>avgTemperatures</code>, and assign value with key <code>tomorrow</code> to <code>tempOfTomorrow</code> in line.
+Use destructuring to obtain the average temperature for tomorrow from the input object <code>AVG_TEMPERATURES</code>, and assign value with key <code>tomorrow</code> to <code>tempOfTomorrow</code> in line.
```

Comment:
```suggestion
Use destructuring to obtain the average temperature for tomorrow from the input object <code>avgTemperatures</code>, and assign value with key <code>tomorrow</code> to <code>tempOfTomorrow</code> in line.
```
This was changes to `avgTemperatures` 5 days ago - I think it should remain like that. If we aren't going to destructure the passed in parameter what's the point of having a parameter.

Reply:
@moT01 It was merged, but it was not supposed to be.  It had been agreed by even the OP of the other PR, that it was fine to leave it as AVG_TEMPERATURES.  There is nothing wrong with the original instruction of using AVG_TEMPERATURES.  It is completely valid JavaScript to use the globallly declared logic or the parameter inside the function.

Reply:
Also, as it was already stated to @lasjorg in a previous comment, if we want to completely change the challenge code and move away from a function, then that discussion should take place in a completely separate issue, so unless there is invalid JavaScript here, please merge.

Reply:
Sounds good